src/surfaces/actionBar/actionBar.jsx

```diff
@@ -245,7 +252,7 @@ class ActionBar extends React.PureComponent {
                                                             iconTable,
                                                         } = item;
 
-                                                        const divide = _.isUndefined(item.divide) ||
+                                                        const divide = isUndefined(item.divide) ||
                                                             item.divide;
```

Comment:
Maybe Not a Blocker because prior code was like this, but this is weird.  So, if `item.divide` is undefined, `divide` will be `true`; otherwise, it is whatever it is.  Is that really the intent?

Also, this just confuses matters further (it's a Boolean, but it defaults to `null`):
https://github.com/saddlebackdev/react-cm-ui/blob/3eed68f2a07caf3a78ec63d3dd84c73d6bc4fbfa/src/dataDisplay/list/listItem.jsx#L7-L21

Reply:
Fixed.

Reply:
Yeah, seems cleaner as of f47470e.  Thanks for fixing that.  Resolving.